scripts/reborn_webui_v2_live_qa/test_run_live_qa.py

```
@@ -1696,27 +1704,137 @@ async def fail_if_waits(_seconds):
+        page, advance = self._fake_sequenced_terminal_page(
+            [
+                {"assistants": [truncated_reply], "errors": []},
+                {
+                    "assistants": [complete_reply],
+                    "errors": [],
+                    "blocks_read_fails": True,
+                },
+            ]
+        )
+
+        async def reveal_complete_reply(_seconds):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `reveal_complete_reply`

Add return type annotation: `None`

(ANN202)